spec/features/shared_examples/sequencing.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
```

Comment:
Has this code been copy pasted from another spec? If so can we reuse it between multiple specs?